include/envoy/config/context_provider.h

```diff
@@ -14,12 +16,45 @@ namespace Config {
  */
 class ContextProvider {
 public:
+  using UpdateCb = std::function<void(absl::string_view)>;
```

Comment:
What does this string_view argument map to?

Reply:
Resource type URL. Will update comment.